destiny/main/bdd/models/__init__.py

```diff
@@ -1,5 +1,5 @@
 from destiny.main.bdd.models.players import Players
-from destiny.main.bdd.models.participant import Participant
+from destiny.main.bdd.models.participants import Participants
```

Comment:
I undertsand why you put Participant in the plural form but event models should be in the plural too